src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -234,6 +240,21 @@ private void EmitLocalAddress(BoundLocal localAccess)
             {
                 _builder.EmitLocalAddress(GetLocal(localAccess));
             }
+            return null;
+        }
+
+        /// <summary>
+        /// May introduce a temp which it will return. (otherwise returns null)
+        /// </summary>
+        private LocalDefinition EmitDupAddress(BoundDup dup, AddressKind addressKind)
+        {
+            if (!HasHome(dup, addressKind != AddressKind.ReadOnly))
```

Comment:
Consider using a named argument for the second parm here. Makes it more readable.